db/migrate/20220215231905_change_opportunity_fields_to_enums.rb

```diff
@@ -0,0 +1,23 @@
+class ChangeOpportunityFieldsToEnums < ActiveRecord::Migration[6.1]
+  def change
+    # we are okay with doing this since we are still on dev :whew: and sync data from SF anyways
+    # done to support using enums for opportunity fields
```

Comment:
Why use enums? I find them annoying when viewing data in the Accounts database. I have to go find what the enum value is. Might just be my problem since I'm using DataGrip and no one else is.

Reply:
I think this actually changed after this migration was created.
I moved the validation (why I'd like to see enums or such in use) to the openstax_salesforce gem: https://github.com/openstax/openstax_salesforce/blob/main/lib/openstax/salesforce/remote/opportunity.rb#L13-L22